packages/sql-drizzle/test/pg.test.ts

```diff
@@ -0,0 +1,53 @@
+import * as Sql from "@effect/sql"
+import * as PgDrizzle from "@effect/sql-drizzle/pg"
+import * as Pg from "@effect/sql-pg"
+import { afterAll, assert, beforeAll, describe, it } from "@effect/vitest"
+import type { StartedPostgreSqlContainer } from "@testcontainers/postgresql"
+import { PostgreSqlContainer } from "@testcontainers/postgresql"
+import { gte } from "drizzle-orm"
+import * as D from "drizzle-orm/pg-core"
+import * as Effect from "effect/Effect"
+import type { Scope } from "effect/Scope"
+import * as Secret from "effect/Secret"
+
+const users = D.pgTable("users", {
+  id: D.serial("id").primaryKey(),
+  name: D.text("name")
+})
+
+describe("PgDrizzle", () => {
+  let pgContainer: StartedPostgreSqlContainer
+  let makeClient: Effect.Effect<Pg.client.PgClient, never, Scope>
+
+  beforeAll(async () => {
+    pgContainer = await new PostgreSqlContainer("postgres:alpine").start()
```

Comment:
using testcontainer, not sure that's the standard way to test db in effect. tell me if it's ok.

Reply:
testcontainer is really great, I usually wrap it in a layer so I can have a TestDb that spins up and down automatically, but this looks fine too